src/asmcnc/comms/router_machine.py

```diff
@@ -1185,35 +1185,36 @@ def correct_rpm(self, requested_rpm, spindle_voltage = None, revert = False):
 
         return rpm_to_set
 
-    def turn_on_spindle(self, rpm=None):
+    def power_on_spindle(self):
```

Comment:
I find it hard to see the difference from "power_on" to "turn_on". This is only used in rare occasions anyway so we could just call turn_on(rpm=0) there. So it is clear that we start with no RPM. No need for a 1-line function.

Reply:
I think it's a very distinct thing from turning on the spindle, it's used for reading data. I see the issue with the naming, but I'd rather change the name than remove the function. But you have the last say ;)